src/main/java/org/opensearch/knn/training/TrainingJobRunner.java

```diff
@@ -149,7 +149,7 @@ private void train(TrainingJob trainingJob) {
 
             ModelMetadata modelMetadata = trainingJob.getModel().getModelMetadata();
             modelMetadata.setState(ModelState.FAILED);
-            modelMetadata.setError(ree.getMessage());
+            modelMetadata.setError("Training job execution was rejected. Node's training queue is at capacity.");
```

Comment:
It would be great to have node name also printed. But we can take in future PRs.

Reply:
agree. Will look into in future.